packages/shared/src/components/modals/UpvotedPopupModal.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import classNames from 'classnames';
+import { ResponsiveModal } from './ResponsiveModal';
+import { ModalProps } from './StyledModal';
+import { Upvote } from '../../graphql/common';
+import { ModalCloseButton } from './ModalCloseButton';
+
+interface UpvotedPopupModalProps extends ModalProps {
+  upvotes?: Upvote[];
+  placeholderAmount?: number;
+}
+
+const containerClassName = 'flex flex-row px-6 mb-3';
+const contentClassName = 'flex flex-col flex-1 ml-4';
+const imageClassName = 'w-12 h-12 rounded-10 bg-pepper-70';
+
+const UpvotedPopupModal: React.FC<UpvotedPopupModalProps> = ({
+  upvotes,
+  placeholderAmount = 5,
+  onRequestClose,
+  ...modalProps
+}) => {
+  const renderPlaceholder = () =>
```

Comment:
Consider refactoring these two components out of this component.
You can create a placeholder component and an upvoter list component and just use them in this one.